doc/releases/release-notes-3.7.rst

```diff
@@ -833,6 +833,13 @@ Drivers and Sensors
   * Migrated :dtcompatible:`holtek,ht16k33` and
     :dtcompatible:`microchip,xec-kbd` from kscan to input subsystem.
 
+* LED
+
+  * Added device completion to LED shell commands and made the ``get_info`` command display
+    colors as strings.
+
+  * Added driver for Lumissil Microsystems (a division of ISSI) IS31FL3194 controller.
```

Comment:
> should we make it a requirement to have mentions to drivers (in particular new ones) in the release notes link to the compatible? 

It would  make sense

Reply:
Added with #76091.